src/core/annotation.js

```diff
@@ -3144,6 +3147,10 @@ class ChoiceWidgetAnnotation extends WidgetAnnotation {
     super(params);
 
     const { dict, xref } = params;
+
+    this.indices = dict.get("I");
+    this.hasIndices = Array.isArray(this.indices) && this.indices.length;
```

Comment:
Since the name suggests that the property should contain *a boolean* value, please update the code to actually make that so.
```suggestion
    this.hasIndices =
      Array.isArray(this.indices) && this.indices.length > 0;
```